topics/proteomics/tutorials/proteome_annotation/tutorial.md

```diff
@@ -0,0 +1,320 @@
+---
+layout: tutorial_hands_on
+
+title: Annotating a protein list identified by LC-MS/MS experiments
+
+zenodo_link: 
+- 'https://zenodo.org/record/2650868'
+
+questions:
+- How to filter out technical contaminants?
+- How to check for tissue-specificity?
+- How to perform enrichment analysis?
+- How to map your protein list to pathways (Reactome)?
+- How to compare your proteome with other studies?
+objectives:
+- Execute a complete annotation pipeline of a protein list identified by LC-MS/MS experiments
+time_estimation: 60M
+key_points: how to annotate a list of proteins ID, Gene Ontology Enrichement analysys, ID conversion 
+
+contributors:
+- vloux
+- combesf
+- davidchristiany
+- yvandenb
+
+---
+
+# Introduction
+{:.no_toc}
+
+[ProteoRE Galaxy instance](http://www.proteore.org) provides necessary tools to execute a whole annotation pipeline of a protein list identified by LC-MS/MS experiments. This activity introduces these tools and guides you through a simple pipeline using some example datasets based on the following study: [Proteomic characterization of human exhaled breath condensate](https://www.ncbi.nlm.nih.gov/pubmed/29189203) by Lacombe *et al., European Journal of Breath, 2018*. 
+
+
+Once identified and/or quantified using a MS-based approach, interpreting the proteome in a sample is an important step to characterize its content in terms of functional properties in order to extend the biological knowledge related to this sample. In this activity, we illustrate the annotation and the exploration of the human exhaled breath condensate (EBC) proteome by performing the following steps:
+> ### Agenda
+>
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Get Input Datasets
+
+For this tutorial, we will use 3 datasets: the list of proteins identified by LC-MS/MS in the exhaled breath condensate (EBC) from Lacombe *et al.* and two others EBC proteomes previously published.
```

Comment:
Hi @combesf, could you please send me the DOIs for the publications you refer to when you say "two other EBC proteomes previously published"

Reply:
Hi @shiltemann 
For Bredberg et al : DOI =  10.1373/clinchem.2011.169235
For Mucilli et al : I found only http://dx.doi.org/10.1016/j.jpba.2014.11.050

is it what you need ? 
Thanks !
Florence

Reply:
yes, that is perfect, thanks!